release/models/p4rt/openconfig-p4rt.yang

```diff
@@ -90,18 +106,16 @@ module openconfig-p4rt {
     }
   }
 
-  augment "/oc-if:interfaces/oc-if:interface/oc-if:config" {
+  grouping p4rt-chassis-config {
     description
-      "Add interface-specific intended configuration for P4RT.";
+      "Config regarding P4RT use cases which corresponds to the global device hardware package";
 
-    uses p4rt-interface-config;
-  }
-
-  augment "/oc-if:interfaces/oc-if:interface/oc-if:state" {
-    description
-      "Add interface-specific applied configuration for P4RT.";
-
-    uses p4rt-interface-config;
+    leaf id {
+      type uint32;
+      description
+        "A unique identifier representing the global hardware device.  This is
```

Comment:
"global hardware device" is interesting wording. Given that this is `chassis-id` should this just be "the overall set of forwarding resources that are controlled by this physical or logical chassis" (to handle multi-chassis)?

Reply:
Also, suggest that this description should include the word "P4RT" somewhere :-)

Reply:
Perhaps:

> An identifier used for the root of oc-p4rt:node-id.  It is used to indicate which oc-p4rt:node-id's belong to the same device. ";